project/Dependencies.scala

```diff
@@ -124,6 +125,8 @@ object Dependencies {
     workbenchGoogle,
     workbenchGoogleTest,
     workbenchMetrics,
-    sam
+    sam,
+
+    sealerate
```

Comment:
just a tiny library for collecting all objects in a sealed trait. See https://github.com/mrvisser/sealerate. `enumeratum` might suffice my need, but I didn't want to extend ADT from `Enum[A]`. `sealerate` is more light weight.